auto-value-moshi/build.gradle

```diff
@@ -1,15 +1,20 @@
+apply plugin: 'net.ltgt.apt'
```

Comment:
So when I use that, it fails on CI due to not finding generated classes - https://travis-ci.org/rharter/auto-value-moshi/builds/456470849?utm_source=github_status&utm_medium=notification

I tried this just in case, and sure enough it did seem to fix it. Any sense on why it only happens in CI?

Reply:
That was because auto-service wasn't in the processor path anymore at that commit.

At a minimum you could revert the `annotationProcessor` → `apt` change, whether using `net.ltgt.apt` or not (it will setup `options.annotationProcessorGeneratedSourcesDirectory`, but this is useless here as those processors only generate resources; see the [project's README](https://github.com/tbroyer/gradle-apt-plugin/blob/master/README.md) to determine whether it's useful for you or not, in this specific case, it's not).